### DISCUSSION TITLE
Update and create big size data

### DISCUSSION THREAD

**Discussion by mehmetuken** [directus/directus#10501] [Archive]
Hi, what is the best way insert or update hourly big size of data (20-50k)?

**Comment by rijkvanzanten** [comment]
Heya! You could use the regular /items endpoint to insert those items. I would recommend batching it in requests of like 1000 items each to make sure you don't blow up the memory usage of the server, or run into any size limits in the web-traffic.

Alternatively, if you don't have to check permissions etc, you could simply insert those records straight into the database through another channel, like a Lambda function.
